### DISCUSSION TITLE
Config.yaml location

### DISCUSSION THREAD

**Discussion by AlexKalopsia** [rommapp/romm#117] [General]
I see that it's possible to add a `config.yaml` file in order to exclude folders from the library. It's unclear to me _where_ the file should live. Is it in the root of the library, or should I mound a volume specifically used for the config?

**Comment by zurdi15** [comment]
Hi, as you can see in the [docker-compose.yml](https://github.com/zurdi15/romm/blob/master/docker/docker-compose.example.yml) example you need to bind the config.yml file to ``/romm/config.yml``. The place where you put it in your host filesystem is up to you.
